tests/baselines/reference/user/prettier.log

```diff
@@ -5,7 +5,7 @@ index.js(138,21): error TS2532: Object is possibly 'undefined'.
 src/cli/util.js(262,64): error TS2339: Property 'length' does not exist on type 'Ignore'.
 src/cli/util.js(335,52): error TS2339: Property 'length' does not exist on type 'Ignore'.
 src/cli/util.js(396,46): error TS2345: Argument of type 'null' is not assignable to parameter of type 'number | undefined'.
-src/cli/util.js(403,39): error TS2339: Property 'grey' does not exist on type 'typeof import("E:/Github/TypeScript/node_modules/chalk/types/index")'.
+src/cli/util.js(403,39): error TS2339: Property 'grey' does not exist on type 'typeof import("/opt/vsts/work/1/s/node_modules/chalk/types/index")'.
```

Comment:
My RE filtering out platform paths doesn;t hit this line because this is an error referencing a path in _our_ node_modules folder and not the test's; that implies this `types: []` alone isn't good enough... (since our `@types/chalk` is still overriding the local `chalk`. Methinks we need to add a `typeRoots` to these tests,